### PR TITLE
XIO as collateral type.

### PR DESCRIPTION
**Pull request by kraikov**

### CODE REVIEW

File: umip-71.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+UMA Accelerator Motivation & Overview:
+
+Over the course of one month (April 1st-30th, 2021), Blockzero will launch three unique campaigns designed to drive education, engagement and adoption of UMA’s various protocols and dapps. Within each phase, there will be an increasing amount of $XIO available to be earned by participants, as well as a varying amount of UMA’s KPI Options.
+
+More specifically, $35K USD worth of XIO and 1,400 of UMA’s KPI Options would be used to incentivize users to acquire and hold synthetic tokens during a “Synthetic Mining” competition during Phase 3 of our Accelerator campaign, with the potential “XIO Yield Dollar” being one of the main assets users would be incentivized to interact with.
+
+We see this as an innovative method of growing both the Blockzero and UMA communities alike, mainly by educating, incentivizing and onboarding users into the UMA ecosystem. A final outline of this program will be released via our Medium on March 25th, 2021.
```
Comment (Reinis-FRP) [inline umip-71.md:33]: [This article](https://medium.com/bombx/defi-ing-gravity-uma-defi-accelerator-ready-for-liftoff-ae8697315fd0)?

File: umip-71.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Motivation
+Adding XIO as a collateral type is required in order to be able to create the XIO Yield Dollar, collateralised by XIO - the primary token of Blockzero Labs. As mentioned above, the XIO Yield Dollar would play a pivotal role in our upcoming UMA DeFi Accelerator (launching on April 1st, 2021).
+
+Blockzero Labs is crypto’s first community-driven token studio. From logo to launch, we build experimental tokens with our community. Our first project, Flashstake (current market cap of $4.8M), launched on January 1st, and the second, AquaFi, is due to arrive in April 2021.
+
+Blockzero Labs has been active for over 2 years, with a current market cap of $8.9M. Liquidity in our XIO/ETH pool on Uniswap has grown every quarter since its inception in May of 2020, and it currently stands at over $6.3M. It is one of the most decentralized pools on the platform, with the largest single holder owning only 6% of the liquidity pool, amongst a total of 661 LP token holders. The XIO/ETH pool’s liquidity ranking on Uniswap is currently #112.
```
Comment (pemulis) [inline umip-71.md:19]: Liquidity on Uniswap looks really solid!

**Review by pemulis — APPROVED** [review]

File: umip-71.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+There are two primary roles for the XIO token within the Blockzero Labs ecosystem. First, the token acts as a governance token which allows community members (Citizens) to vote on important topics and decisions. Second, the XIO token acts as a redemption mechanism for future Blockzero assets. Holding XIO gives users the ability to access and claim alternative tokens. In the future, the XIO token will also be the primary governance token for the Blockzero Index (Treasury).
```
Comment (akhanubis) [inline umip-71.md:21]: Will sponsors with XIO locked in EMPs be able to vote and interact with the redemption mechanism?

File: umip-71.md (hunk 1 above)
Comment (mhairimca) [reply umip-71.md:33]: I think this might be a bit too specific for an UMIP.

File: umip-71.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+More specifically, $35K USD worth of XIO and 1,400 of UMA’s KPI Options would be used to incentivize users to acquire and hold synthetic tokens during a “Synthetic Mining” competition during Phase 3 of our Accelerator campaign, with the potential “XIO Yield Dollar” being one of the main assets users would be incentivized to interact with.
+
+We see this as an innovative method of growing both the Blockzero and UMA communities alike, mainly by educating, incentivizing and onboarding users into the UMA ecosystem. A final outline of this program will be released via our Medium on March 25th, 2021.
+
+## Technical Specification
+To accomplish this upgrade, two changes need to be made:
+
+- The XIO address, 0x0f7F961648aE6Db43C75663aC7E5414Eb79b5704, needs to be added to the collateral currency whitelist introduced in UMIP-8.
+- A final fee of 1500 $XIO needs to be added in the Store contract. This is approximately $500 at current XIO rates.
```
Comment (smb2796) [inline umip-71.md:39]: Final fee seems reasonable

**Review by smb2796 — APPROVED** [review]
Looks like any blocking issues were resolved. Going to merge in and move to Last Call

**Pull request merged**